src/index.tsx

```diff
@@ -40,32 +42,52 @@ if (window.location.hostname !== 'localhost') {
     integrations: [
       // Sentry.browserTracingIntegration(),
       // Sentry.replayIntegration(),
-      Sentry.httpClientIntegration({
+      httpClientIntegration({
         failedRequestStatusCodes: [
           [400, 428],
           // i.e no 429s
           [430, 599],
         ],
 
-        failedRequestTargets: [/^(?!.*\.?(alchemy|snapshot)\.(com|org)).*$/],
+        // denyUrls: ['alchemy.com', 'snapshot.org'],
```

Comment:
is commenting this out intentional?

Reply:
Absolutely not, forgotten monkey 🐒  055843f79e